Makefile.custom.mk

```diff
@@ -31,6 +31,10 @@ test-inhibitions: install-tools template-chart
 	./test/hack/bin/get-inhibition.sh
 	cd test/hack/checkLabels; go run main.go
 
+test-opsrecipes: install-tools template-chart
+    # Check if opsrecipes are valid
```

Comment:
Indentation is wrong (or github show it wrong)

Reply:
tab vs spaces... fixed.